lib/puppet/util/plist.rb

```diff
@@ -60,7 +61,7 @@ def parse_plist(plist_data, file_path = '')
 
       begin
         plist_obj = new_cfpropertylist(:data => plist_data)
-      rescue CFFormatError => e
+      rescue CFFormatError, NoMethodError => e
```

Comment:
To be honest, this is probably(?) a bug in `cfpropertylist` and it should catch the `NoMethodError` and throw `CFFormatError`. (I haven't dug into the code, though.) So in that sense both `CFFormatError` and `NoMethodError` are catching the same thing.

I'm happy to do it either way, though.